style.css

```diff
@@ -0,0 +1,128 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  background: #e5e5e5;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+}
+
+.main {
+  background: url('./images/header-background-shape-desktop.svg');
+  min-height: 100vh;
+  width: 100vw;
+  margin: 0;
+  padding-top: 72px;
+}
+
+.toolbar {
+  position: fixed;
+  margin: 0;
+  height: 72px;
+  width: 95vw;
+  padding: 0 24px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+}
+
+.logo-wrapper {
+  height: 20px;
+  width: 76px;
+}
+
+.menu-icon {
+  height: 16px;
+  width: 17px;
+}
+
+.header {
+  background: url('./images/Header-bg.svg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  height: 85vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.banner {
+  width: 63.888%;
+}
+
```

Comment:
**REQUIRED:** The banner has much more space on the left and right because of setting its width to 63.888%, instead of it use margin and padding.

![image](https://user-images.githubusercontent.com/77872355/129120760-5e4755a3-e48b-46fe-940d-010285e04638.png)
![image](https://user-images.githubusercontent.com/77872355/129120835-a9f0e05f-e9ed-4184-9b64-7dc0465abc11.png)


**OPTIONAL:** There is a little difference between font color of this section, it would be good idea if you change it.